try {
    console.log("Tentar login")

    login()

    console.log("Tentativa bem sucedida.")
} catch (error) {
    console.log("Erro!")
}